examples/abi/src/lib.rs

```diff
@@ -0,0 +1,45 @@
+use near_sdk::near_abi;
+
+#[near_abi]
+mod abi {
+    use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+    use near_sdk::near_bindgen;
+    use near_sdk::schemars::JsonSchema;
+    use near_sdk::serde::{Deserialize, Serialize};
+
+    #[derive(JsonSchema, Serialize, Deserialize)]
+    pub struct Pair(u32, u32);
+
+    #[derive(JsonSchema, Serialize, Deserialize)]
+    pub struct DoublePair {
+        first: Pair,
+        second: Pair,
+    }
+
+    #[near_bindgen]
+    #[derive(Default, BorshDeserialize, BorshSerialize)]
+    pub struct Adder {}
+
+    #[near_bindgen]
+    impl Adder {
+        pub fn add(&self, a: Pair, b: Pair) -> Pair {
+            sum_pair(&a, &b)
+        }
+
+        pub fn add_callback(
+            &self,
+            #[callback_unwrap] a: DoublePair,
+            #[callback_unwrap] b: DoublePair,
+            #[callback_vec] others: Vec<DoublePair>,
+        ) -> DoublePair {
+            vec![b].iter().chain(others.iter()).fold(a, |acc, el| DoublePair {
```

Comment:
nit:
```suggestion
            Some(b).iter().chain(others.iter()).fold(a, |acc, el| DoublePair {
```
or just using array instead of vec which would allocate